<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDialog</class>
 <widget class="QDialog" name="ImportDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Photos</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>394</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="rbCopy">
        <property name="text">
         <string>Copy</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbMove">
        <property name="text">
         <string>Move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>394</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmMain">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QFrame" name="frmSource">
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>-1</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="cbIncludeSubdirs">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Include subdirectories</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="trvwSource"/>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frmTileView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="Widgets::TileView" name="mCfvPhotos" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frmToolBar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>16</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>177</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>16</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frmDestination">
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeView" name="trvwDestination"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
     <zorder>frmSource</zorder>
     <zorder>frmDestination</zorder>
     <zorder>trvwDestination</zorder>
     <zorder>trvwSource</zorder>
     <zorder>frmTileView</zorder>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmButtons">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>408</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnImport">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Import</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widgets::TileView</class>
   <extends>QWidget</extends>
   <header>widgets/tileview.h</header>
   <container>1</container>
   <slots>
    <slot>selectAll()</slot>
    <slot>clearSelection()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>946</x>
     <y>759</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnImport</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1031</x>
     <y>759</y>
    </hint>
    <hint type="destinationlabel">
     <x>623</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trvwSource</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>ImportDialog</receiver>
   <slot>onSourceDirClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trvwDestination</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>ImportDialog</receiver>
   <slot>onDestinationDirClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>1048</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbCopy</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>onImportModeCopy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbMove</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>onImportModeMove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>746</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbAdd</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>onImportModeAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbIncludeSubdirs</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>onIncludeSubdirs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSourceDirClicked(QModelIndex)</slot>
  <slot>onDestinationDirClicked(QModelIndex)</slot>
  <slot>onImportModeCopy()</slot>
  <slot>onImportModeMove()</slot>
  <slot>onImportModeAdd()</slot>
  <slot>onIncludeSubdirs()</slot>
 </slots>
</ui>
